src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -113,6 +114,15 @@ public void disconnected(ExecutorDriver driver) {
 
     @Override
     public void launchTask(final ExecutorDriver driver, final TaskInfo task) {
+        // "task id beginning with "yarn" is a mesos task for yarn container
+        if (task.getTaskId().getValue().startsWith("yarn")) {
```

Comment:
@DarinJ Can you please elaborate more? Are you saying that if Myriad chooses to launch Myriad executor and NM inside docker container, the launch of pseudo tasks may not resize the docker container?